src/test/regress/expected/multi_sequence_default.out

```diff
@@ -130,72 +130,6 @@ ALTER SEQUENCE seq_0 AS bigint;
 ERROR:  Altering a distributed sequence is currently not supported.
 ALTER SEQUENCE seq_0_local_table AS bigint;
 ERROR:  Altering a distributed sequence is currently not supported.
--- we can change other things like increment
```

Comment:
it is kind of sad that we are dropping these tests. Do you see a way to somehow keep them? 

Or at least understand what they test and maybe add something similar that covers the same?

Reply:
Since we make sequences objects distributed, we won't be able to test those below. So, not covering them looks ok to me.